backend/src/controllers/email.ts

```diff
@@ -96,6 +96,45 @@ export const enableEmail = async (req: Request, res: Response) => {
     }
 }
 
+export const updateEmailTemplate = async (req: Request, res: Response) => {
+    // only admin can save templates
+    let userClaims = (req as CustomRequest).firebase
+    if (!verifyIfUserAdmin(userClaims)) {
+        Logger.error({
+            message: 'User is not admin',
+            moduleName,
+            function: 'enableEmail',
+            statusCode: 403,
+        })
+        return res.status(403).json({ error: 'forbidden' })
+    }
+
+    // TODO validate template body
+    const template = req.body.template
+    const subject = template?.subject
+    const content = template?.content
+    if (!template || !subject || !content) {
+        Logger.error({
+            moduleName,
+            message: 'No template provided',
+            statusCode: 400,
+        })
+        return res.status(400).json({ error: 'bad_request' })
+    }
+
+    await db.collection(nodeConfig.get('collections.email')).doc('template').update({
```

Comment:
In Firestore all the cron-related data will stay under cron collection but I can see that you created a new email collection which is not required. So can you move it under the cron collection?